Header Header

cmu_unet/line far_left
cmu_unet/line left
cmu_unet/line right
cmu_unet/line far_right

# Although there are four lines in this message, some of them may be empty if they do not exist in the environment.
# It is also possible some lines are discarded because there is no room for them in the message
# It is the responsibility of the driving controller to handle these lines.